lib/transactionsFilter/emitInstantLockToFilterCollectionFactory.js

```diff
@@ -0,0 +1,22 @@
+const { InstantLock } = require('@dashevo/dashcore-lib');
+
+/**
+ * @param {BloomFilterEmitterCollection} bloomFilterEmitterCollection
+ * @return {testBlockEventToFilterCollection}
+ */
+function emitInstantLockToFilterCollectionFactory(bloomFilterEmitterCollection) {
+  /**
+   * Emit `islock` event to bloom filter collection
+   *
+   * @param {Buffer} rawTransactionLock
+   */
+  function testBlockEventToFilterCollection(rawTransactionLock) {
+    const transactionLock = new InstantLock(rawTransactionLock);
```

Comment:
You need to parse out the ISlock either here or where the zmq event happens, because the `rawTransactionLockSig` will have both the `rawTx` and concatenated to it the isLock with signature in it
https://github.com/dashpay/dash/blob/master/src/zmq/zmqpublishnotifier.cpp#L321-L322

Reply:
Fixed and changed test mocks

Reply:
Thanks, looking great now!